tgstation.dme

```diff
@@ -1266,7 +1266,6 @@
 #include "code\modules\events\abductor.dm"
 #include "code\modules\events\alien_infestation.dm"
 #include "code\modules\events\anomaly.dm"
-#include "code\modules\events\anomaly_bluespace.dm"

```

Comment:
?

Reply:
probably forgot to re-check shit when I pulled